sentry-ruby/lib/sentry/client.rb

```diff
@@ -272,6 +273,53 @@ def send_event(event, hint = nil)
       raise
     end
 
+    # Send an envelope with batched logs
+    # @param log_events [Array<LogEvent>] the log events to be sent
+    # @api private
+    # @return [void]
+    def send_logs(log_events)
+      envelope = Envelope.new(
+        event_id: Sentry::Utils.uuid,
+        sent_at: Sentry.utc_now.iso8601,
+        dsn: configuration.dsn,
+        sdk: Sentry.sdk_meta
+      )
+
+      discarded_logs = []
```

Comment:
let's just keep a count since we don't need the discarded events to be slightly more memory efficient

Reply:
@sl0thentr0py done